### DISCUSSION TITLE
We struggled to comprehend the VSGV and DSGV results files.

### DISCUSSION THREAD

**Discussion by LingXu2018** [korem-lab/SGVFinder2#31] [General]
We are seeking clarification on the meaning of the values (both positive and negative) in the vsgv file. Additionally, could you explain the 'NAs' in the dsgv file? Lastly, what do the column names and row names represent?

**Comment by allen-zhan340** [comment]
> We are seeking clarification on the meaning of the values (both positive and negative) in the vsgv file. Additionally, could you explain the 'NAs' in the dsgv file? Lastly, what do the column names and row names represent?

Have you solved this problem?  I also want to know the meaning of the resulting files?

**Reply by talkorem** [reply]
vsgv values are in units of standard deviation from the mean coverage of the genome.
There are NAs in the dsgv file because the genome was not detected in sufficient coverage.
Column names X_Y, for a bin size of B, mean genome position of XB to YB-1.
